### ISSUE TITLE
Chat input is blocked and spinner is shown on user avatar when synced chat is shown on mobile 

### ISSUE THREAD

**Issue by churik** [status-im/status-react#14893]
# Bug Report

## Problem 
Likely a consequence of https://github.com/status-im/status-mobile/pull/14878 
Currently if sending a message from a desktop, it is shown on a paired mobile device with blocked chat input and the endless spinner is displayed instead of the user avatar

#### Expected behavior
not sure; as it is a consequence of another issue, this situation shouldn't be possible when the sync of contacts is fixed

#### Actual behavior
<img width="243" alt="Monosnap 2023-01-25 17-35-00" src="https://user-images.githubusercontent.com/4557972/214621720-fae1cd05-5081-47ad-842e-7e549f7fb8c4.png">

### Reproduction
**Prerequisites:** mobile and desktop are paired using 

1) Desktop: send any message to 1-1 chat

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS
- Desktop version: 0.9.0 rc7

**Comment by cammellos** [comment]
@churik this should be  fixed in develop, could you please check?
Thank you

**Comment by cammellos** [comment]
I believe this is fixed, closing for now, @churik feel free to reopen if necessary
